lib/events.js

```diff
@@ -416,8 +417,10 @@ EventEmitter.prototype.removeListener =
 
         if (position === 0)
           list.shift();
-        else
+        else {
+          spliceOne = spliceOne || require('internal/util').spliceOne;
```

Comment:
It would be better to write
```js
if (spliceOne === undefined) spliceOne = require('internal/util').spliceOne;
```
Otherwise there is a type conversion while evalutating if `spliceOne` is truthy.

Reply:
Same code in https://github.com/nodejs/node/blob/master/lib/events.js#L74
Should I rewrite them all?

Reply:
I would keep it as is to reduce the churn.

Reply:
Existing code is tricky but when adding new code and we're certain that something is either defined or `undefined` then a strict comparison is always better. 

(Also, for example, domain can be null or undefined and maybe even other falsey values that we don't test for.)